plugins/inputs/mqtt_consumer/README.md

```diff
@@ -194,5 +194,16 @@ sensors,site=CLE,version=v1,device_name=device5 temp=390,rpm=45.0,ph=1.45
 
 - example when [[inputs.mqtt_consumer.topic_parsing]] is set
 
+- when [[inputs.internal]] is set:
+  - bytes_received (int): count of the number of bytes have been received from incoming messages
+  - messages_received (int): count of the number of messages that have been received from mqtt
+  
+This will result in the following metric:
+
+```text
+internal_mqtt_consumer,host=pop-os,version=1.24.0-2a266e53
+messages_received=622i,bytes_received=37942i 1657282270000000000
+```
```

Comment:
Should we maybe also add the `internal` metric to `plugins/inputs/internal/README.md`?

Reply:
We took the example from the [GitHub input](https://github.com/influxdata/telegraf/tree/master/plugins/inputs/github#example-output).